time="2022-02-14T13:47:54+02:00" level=error msg="Unsuccessful 1FA authentication attempt by user 'fakeuser': user not found" method=POST path=/api/firstfactor remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-02-14T13:47:54+02:00" level=error msg="Unsuccessful 1FA authentication attempt by user 'fakeuser@example.com': user not found" method=POST path=/api/firstfactor remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-02-14T13:49:12+02:00" level=error msg="Unsuccessful 1FA authentication attempt by user 'realuser'" method=POST path=/api/firstfactor remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-02-14T13:49:12+02:00" level=error msg="Unsuccessful 1FA authentication attempt by user 'realuser@example.com'" method=POST path=/api/firstfactor remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-03-30T14:28:52+02:00" level=debug msg="Successful 1FA authentication attempt made by user 'realuser'" method=POST path=/api/firstfactor remote_ip=127.0.0.1
time="2022-02-16T11:48:16+02:00" level=error msg="Unsuccessful Duo authentication attempt by user 'realuser': duo auth result: deny, status: deny, message: Login request denied." method=POST path=/api/secondfactor/duo remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-02-16T11:48:16+02:00" level=error msg="Unsuccessful Duo authentication attempt by user 'realuser@example.com': duo auth result: deny, status: deny, message: Login request denied." method=POST path=/api/secondfactor/duo remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-03-30T14:28:58+02:00" level=debug msg="Successful Duo authentication attempt made by user 'realuser'" method=POST path=/api/secondfactor/duo remote_ip=127.0.0.1
time="2022-02-16T12:24:19+02:00" level=error msg="Unsuccessful TOTP authentication attempt by user 'realuser'" method=POST path=/api/secondfactor/totp remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-02-16T12:24:19+02:00" level=error msg="Unsuccessful TOTP authentication attempt by user 'realuser@example.com'" method=POST path=/api/secondfactor/totp remote_ip=1.1.1.1 stack="longstacktrace"
time="2022-03-30T14:28:58+02:00" level=debug msg="Successful Duo authentication attempt made by user 'realuser'" method=POST path=/api/secondfactor/duo remote_ip=127.0.0.1
{"level":"error","method":"POST","msg":"Unsuccessful 1FA authentication attempt by user 'fakeuser': user not found","path":"/api/firstfactor","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":177,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_firstfactor.go","Line":52,"Name":"FirstFactorPost.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:31:49+02:00"}
{"level":"error","method":"POST","msg":"Unsuccessful 1FA authentication attempt by user 'fakeuser@example.com': user not found","path":"/api/firstfactor","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":177,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_firstfactor.go","Line":52,"Name":"FirstFactorPost.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:31:49+02:00"}
{"level":"error","method":"POST","msg":"Unsuccessful 1FA authentication attempt by user 'realuser'","path":"/api/firstfactor","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":181,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_firstfactor.go","Line":60,"Name":"FirstFactorPost.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:31:58+02:00"}
{"level":"error","method":"POST","msg":"Unsuccessful 1FA authentication attempt by user 'realuser@example.com'","path":"/api/firstfactor","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":181,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_firstfactor.go","Line":60,"Name":"FirstFactorPost.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:31:58+02:00"}
{"level":"debug","method":"POST","msg":"Successful 1FA authentication attempt made by user 'realuser'","path":"/api/firstfactor","remote_ip":"127.0.0.1","time":"2022-03-30T14:24:18+02:00"}
{"level":"error","method":"POST","msg":"Unsuccessful Duo authentication attempt by user 'realuser': duo auth result: deny, status: deny, message: Login request denied.","path":"/api/secondfactor/duo","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":177,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_sign_duo.go","Line":74,"Name":"SecondFactorDuoPost.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/require_first_factor.go","Line":15,"Name":"RequireFirstFactor.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:32:13+02:00"}
{"level":"error","method":"POST","msg":"Unsuccessful Duo authentication attempt by user 'realuser@example.com': duo auth result: deny, status: deny, message: Login request denied.","path":"/api/secondfactor/duo","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":177,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_sign_duo.go","Line":74,"Name":"SecondFactorDuoPost.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/require_first_factor.go","Line":15,"Name":"RequireFirstFactor.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:32:13+02:00"}
{"level":"debug","method":"POST","msg":"Successful Duo authentication attempt made by user 'realuser'","path":"/api/secondfactor/duo","remote_ip":"127.0.0.1","time":"2022-03-30T14:26:22+02:00"}
{"level":"error","method":"POST","msg":"Unsuccessful TOTP authentication attempt by user 'realuser'","path":"/api/secondfactor/totp","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":181,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_sign_totp.go","Line":41,"Name":"SecondFactorTOTPPost"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/require_first_factor.go","Line":15,"Name":"RequireFirstFactor.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:32:54+02:00"}
{"level":"error","method":"POST","msg":"Unsuccessful TOTP authentication attempt by user 'realuser@example.com'","path":"/api/secondfactor/totp","remote_ip":"1.1.1.1","stack":[{"File":"github.com/authelia/authelia/v4/internal/handlers/response.go","Line":181,"Name":"markAuthenticationAttempt"},{"File":"github.com/authelia/authelia/v4/internal/handlers/handler_sign_totp.go","Line":41,"Name":"SecondFactorTOTPPost"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/require_first_factor.go","Line":15,"Name":"RequireFirstFactor.func1"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go","Line":52,"Name":"AutheliaMiddleware.func1.1"},{"File":"github.com/fasthttp/router@v1.4.5/router.go","Line":414,"Name":"(*Router).Handler"},{"File":"github.com/authelia/authelia/v4/internal/middlewares/log_request.go","Line":14,"Name":"LogRequestMiddleware.func1"},{"File":"github.com/valyala/fasthttp@v1.32.0/server.go","Line":2298,"Name":"(*Server).serveConn"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":223,"Name":"(*workerPool).workerFunc"},{"File":"github.com/valyala/fasthttp@v1.32.0/workerpool.go","Line":195,"Name":"(*workerPool).getCh.func1"},{"File":"runtime/asm_amd64.s","Line":1581,"Name":"goexit"}],"time":"2022-02-16T12:32:54+02:00"}
{"level":"debug","method":"POST","msg":"Successful Duo authentication attempt made by user 'realuser'","path":"/api/secondfactor/duo","remote_ip":"127.0.0.1","time":"2022-03-30T14:26:22+02:00"}
